subsys/bluetooth/services/fast_pair/crypto/fp_crypto_common.c

```diff
@@ -10,7 +10,7 @@
 #include <zephyr/sys/byteorder.h>
 
 #include "fp_crypto.h"
-
+#include "fp_storage.h"
```

Comment:
This dependency is already included by the crypto header

Reply:
I think we still should explicitly include it if we use definition from storage here (I would not assume that storage header will always be included by the crypto header)